pages/common/base64.md

```diff
@@ -0,0 +1,11 @@
+# base64
+
+> Encode and decode using Base64 representation
+
+- Encode using Base64:
+
+`echo {{any content}} | base64 `
```

Comment:
I suppose this would always be text, in order for you to type it, right? In that case I'd suggest `{{plain_text}}` and `{{base64_text}}`.